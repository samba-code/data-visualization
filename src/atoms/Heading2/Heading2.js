import React from "react";
import styled from "styled-components";
import { rem, darken } from "polished";

import PropTypes from "prop-types";

const StyledHeading2 = styled.h2`
  font-size: ${rem("22px")};
  line-height: ${rem("26px")};
  color: ${(props) => darken(0.2, props?.theme?.secondary ?? "black")};
  font-weight: bold;
  font-family: "museo-sans", sans-serif;
  font-weight: 700;
  margin-top: 35px;
`;

const Heading2 = ({ children }) => <StyledHeading2>{children}</StyledHeading2>;

Heading2.propTypes = {
  children: PropTypes.string,
};

export default Heading2;
