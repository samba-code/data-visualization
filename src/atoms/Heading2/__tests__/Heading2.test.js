import React from "react";
import { render } from "enzyme";

import Heading2 from "../Heading2";

describe("Heading2", () => {
  it("should match snapshot", () => {
    const wrapper = render(<Heading2 />);
    expect(wrapper).toMatchSnapshot();
  });
});
