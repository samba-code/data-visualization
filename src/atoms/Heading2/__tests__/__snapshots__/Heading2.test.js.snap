// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading2 should match snapshot 1`] = `
<h2
  class="Heading2__StyledHeading2-sc-1oorg19-0 hpuRNy"
/>
`;
